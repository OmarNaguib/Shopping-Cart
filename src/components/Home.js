export default function Home() {
  return (
    <div className="home">
      <div className="home-message">
        <div className="text">
          <span>Athletes are cool,</span>
          <span>But some are just... </span>
          <span className="special-text">Exceptional.</span>
        </div>
      </div>
    </div>
  );
}
