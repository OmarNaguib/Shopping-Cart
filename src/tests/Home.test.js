import React from "react";
import { render } from "@testing-library/react";
import Home from "../components/Home";
import { HashRouter, Route, Routes } from "react-router-dom";

describe("Home page", () => {
  it("renders correctly", () => {
    const { container } = render(
      <HashRouter>
        <Routes>
          <Route path="/" element={<Home />}></Route>
        </Routes>
      </HashRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
