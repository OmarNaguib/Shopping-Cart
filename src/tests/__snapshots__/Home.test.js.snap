// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
<div>
  <div
    class="home"
  >
    <div
      class="home-message"
    >
      <div
        class="text"
      >
        <span>
          Athletes are cool,
        </span>
        <span>
          But some are just... 
        </span>
        <span
          class="special-text"
        >
          Exceptional.
        </span>
        <a
          class=""
          href="#/shop"
        >
          Get Ready!
        </a>
      </div>
    </div>
  </div>
</div>
`;
